### PR TITLE
add SealLoading icon, CardTitle, LoadingCard

### PR DESCRIPTION
**Pull request by MixailE**

### CODE REVIEW

**Review by smyzin — APPROVED** [review]
Lgtm

File: src/components/LoadingCard.tsx [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+export default function ({
+  title,
+  subtitle,
+}: {
+  title: string
+  subtitle: string
+}) {
+  const { cardColor } = useContext(CardContext)
+  return (
```
Comment (T-Damer) [inline src/components/LoadingCard.tsx:34]: Let's add blank line between hook and return

File: src/components/Card.tsx [hunk 2]
```diff
@@ -114,15 +115,17 @@ export default function ({
...
-      className={cardContainer(shadow, color, onlyWrap, thin, small, nospace)}
-    >
-      {children}
-      {!!spinner && (
-        <div className={spinnerBox}>
-          <ArcText text={spinner} />
-        </div>
-      )}
-    </div>
+    <CardContext.Provider value={{ cardColor: color }}>
```
Comment (T-Damer) [inline src/components/Card.tsx:118]: Maybe we can use import statement like:
```TS
import {Provider} from 'components/CardContext'
```
?

File: src/models/CardColor.ts [hunk 3]
```diff
@@ -0,0 +1,8 @@
+type CardColor =
```
Comment (T-Damer) [inline src/models/CardColor.ts:1]: Why don't you use `models/Color.ts`? Simply extend it

File: src/icons/SealLoading.tsx [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+        stroke-width="2"
+        stroke-linecap="round"
+      />
+      <path
+        d="M91 83C91 77.4772 86.5228 73 81 73C75.4772 73 71 77.4772 71 83C71 88.5228 75.4772 93 81 93"
+        stroke="currentColor"
+        stroke-width="2"
+        stroke-linecap="round"
+      >
+        <animateTransform
```
Comment (T-Damer) [inline src/icons/SealLoading.tsx:69]: SVG animations may be incorrect in some cases, are you sure it works fine on PCs, phones and tablets?

File: src/icons/SealLoading.tsx [hunk 5]
```diff
@@ -0,0 +1,129 @@
+import CardColor from 'models/CardColor'
+import classnames from 'classnames/tailwind'
+import colorToCardDropShadow from 'helpers/colorToCardDropShadow'
+import colorToTextColor from 'helpers/colorToTextColor'
+
+export default function ({ color }: { color: CardColor }) {
+  const textColor = colorToTextColor(color)
+  const shadowColor = colorToCardDropShadow(color)
+  return (
```
Comment (T-Damer) [inline src/icons/SealLoading.tsx:9]: Let's add a blank line between `const` and `return`

File: src/helpers/colorToCardDropShadow.ts [hunk 6]
```diff
@@ -0,0 +1,17 @@
+import { dropShadow } from 'classnames/tailwind'
+import CardColor from 'models/CardColor'
+
+export default function (color: CardColor) {
```
Comment (T-Damer) [inline src/helpers/colorToCardDropShadow.ts:4]: I think it's `colorToDropShadow.ts`, simply extend `models/Color.ts` and extend `helpers/colorToDropShadow.ts` so we don't create new files

**Review by T-Damer — CHANGES_REQUESTED** [review]
Mostly good, only few changes needed :)

File: src/components/Card.tsx (hunk 2 above)
Comment (MixailE) [reply src/components/Card.tsx:118]: This export won't work :(

File: src/icons/SealLoading.tsx (hunk 4 above)
Comment (MixailE) [reply src/icons/SealLoading.tsx:70]: Corrected animateTransform similarly to ArcText

**Review by T-Damer — APPROVED** [review]
Lgtm

**Pull request merged**